sbt/src/sbt-test/source-dependencies/macro-annotation/macros/src/main/scala/Macros.scala

```diff
@@ -3,24 +3,24 @@ import scala.language.experimental.macros
 import scala.annotation.StaticAnnotation
 
 object HelloMacro {
-  def impl(c: Context)(annottees: c.Tree*): c.Tree = {
+  def impl(c: Context)(annotatees: c.Tree*): c.Tree = {
     import c.universe._
 
-    annottees match {
+    annotatees match {
       case (classDecl: ClassDef) :: Nil =>
-        val q"$mods class $name[..$tparams] $ctorMods(...$paramss) extends { ..$earlydefns } with ..$bases { $self => ..$body }" = classDecl
+        val q"$mods class $name[..$tparams] $ctorMods(...$params) extends { ..$earlydefns } with ..$bases { $self => ..$body }" = classDecl
         q"""
-        case class $name(...$paramss) extends ..$bases {
+        case class $name(...$params) extends ..$bases {
```

Comment:
I can't tell why you have an extra `s` here. It could be intentional, but, I'm betting it isn't.
Note: if it is intentional, you could improve readability by expanding the `s` into something more...